src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -288,11 +291,67 @@ export default class DefaultDeviceController
       this.logger.error('Audio input device cannot be undefined');
       return;
     }
+
     if (isAudioTransformDevice(device)) {
       // N.B., do not JSON.stringify here — for some kinds of devices this
       // will cause a cyclic object reference error.
       this.logger.info(`Choosing transform input device ${device}`);
+
+      /*
+       * This block of code is a workaround for a Chromium bug:
+       * https://bugs.chromium.org/p/chromium/issues/detail?id=1173656
+       *
+       * In short: if we are about to select an audio device with a transform, which we assume for
+       * safety's sake uses AudioWorklet, we recreate the audio context and the nodes that
+       * are linked to it.
+       *
+       * This causes Chrome to rejig its buffers and the second context works correctly.
+       *
+       * This is theoretically worse for performance, but in practice it is fine.
+       *
+       * This is not safe in the general case: an application that already
+       * retrieved the audio context in order to build an audio graph for some other purpose
+       * will fail at this point as we pull the context out from under it.
+       *
+       * An application that always uses the supplied context in an
+       * `AudioTransformDevice.createAudioNode` call should work correctly.
+       *
+       * If you are confident that your application does not use AudioWorklet, does not run in
+       * an un-fixed Chromium version, or will never be used with sample-rate-switching Bluetooth
+       * devices, you can leave this hack disabled.
```

Comment:
How do one disable this? Do you mean they create their own device controller implementation without this?

Reply:
See conversation above about passing in a new `ExtendedBrowserBehavior` instance! I'll leave a note.